ledger/db/src/lib.rs

```diff
@@ -1358,22 +1367,11 @@ mod ledger_db_test {
                 (0..5).map(|_i| KeyImage::from(rng.next_u64())).collect();
 
             let block_contents = BlockContents::new(key_images.clone(), outputs);
-            assert_eq!(last_block.version, BLOCK_VERSION);
-
-            let invalid_block = Block::new_with_parent(
-                last_block.version + 1,
```

Comment:
this test is not possible to run anymore because `BlockVersion::MAX + 1` is enforced not to exist as a `BlockVersion`

Reply:
we could try testing that getting an `ArchiveBlock` that has `BlockVersion::MAX + 1` fails with a block version error